scripts/Muon/GUI/Common/plot_widget/plot_widget_view.py

```diff
@@ -22,6 +22,7 @@ def warning_popup(message):
     def __init__(self, parent=None):
         super().__init__(parent=parent)
         self.setupUi(self)
+        self.plot_diff_checkbox.setVisible(False)
```

Comment:
We should set this somewhere in the `__init__` of the presenter using an `enable_plot_diff method` as mentioned above.